docs/data-sources/http.md

```diff
@@ -32,22 +45,21 @@ data "http" "example" {
 }
 ```
 
-## Argument Reference
+<!-- schema generated by tfplugindocs -->
+## Schema
 
-The following arguments are supported:
+### Required
 
-* `url` - (Required) The URL to request data from. This URL must respond with
-  a `200 OK` response and a `text/*` or `application/json` Content-Type.
+- `url` (String) The URL for the request. Supported schemes are `http` and `https`.
 
-* `request_headers` - (Optional) A map of strings representing additional HTTP
-  headers to include in the request.
+### Optional
 
-## Attributes Reference
+- `request_headers` (Map of String) A map of request header field names and values.
 
-The following attributes are exported:
+### Read-Only
+
+- `body` (String) The response body returned as a string.
+- `id` (String) The ID of this resource.
+- `response_headers` (Map of String) A map of response header field names and values.
 
-* `body` - The raw body of the HTTP response.
 
-* `response_headers` - A map of strings representing the response HTTP headers.
-  Duplicate headers are contatenated with `, ` according to
-  [RFC2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2)
```

Comment:
Should we still be capturing this additional RFC detail in the attribute description?

Reply:
Error on my part. Have reinstated.